test/test.js

```diff
@@ -917,6 +921,28 @@ async function run () {
     const queryParams = JSON.stringify(qs.parse('foo=bar&foo=baz&abc=qux'))
     t.equal(firstProxyPrefix.body, `this is "variable-api" endpoint with id 123 and query params ${queryParams}`)
   })
+
+  test('keep the query params on proxy without params in prefix', { only: true }, async t => {
```

Comment:
If I run this test with the old code, it passes.

Reply:
You are right, it passes. Var `dest` from `reply.from` don't have query params in value (https://github.com/fastify/fastify-http-proxy/blob/master/index.js#L341), but they passed to upstream anyway. I suppose `reply.from` copy query params underhood

I retest my project again and it's all okay, query params passed in current version. So, I close pull request. Anyway, thank you for the review, and sorry for wasting time.